unzip/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
-set(HEADERS)
 set(CPPS unzip.cpp)
 
 add_library(unzip STATIC ${HEADERS} ${CPPS})
 
-target_link_libraries(unzip ZLIB::ZLIB)
+target_link_libraries(unzip PUBLIC ZLIB::ZLIB)
```

Comment:
this should be `PRIVATE`, as consumers should not use the `ZLIB::ZLIB` target directly